tests/common/libtest.sh

```diff
@@ -388,8 +388,8 @@ EOF
             echo "Provides: $arg" >> $spec;;
         conflicts)
             echo "Conflicts: $arg" >> $spec;;
-        post_interp)
-            post_interp="$arg"; post="$1"; shift;;
+        post_args)
+            post_args="$arg"; post="$1"; shift;;
```

Comment:
Minor: can we keep these separate? I.e. require `post_args` and `post` to be passed on their own, rather than slurp the third arg as part of it? Would make its usage simpler to read.